kvs/src/monitor/monitoring.cpp

```diff
@@ -92,8 +92,16 @@ int main(int argc, char *argv[]) {
   unsigned ebs_node_number;
   // keep track of the keys' access by worker address
   map<Key, map<Address, unsigned>> key_access_frequency;
+  // keep track of the hot keys' access by worker address
+  map<Key, map<Address, unsigned>> hot_key_access_frequency;
+  // keep track of the cold keys' access by worker address
+  map<Key, map<Address, unsigned>> cold_key_access_frequency;
   // keep track of the keys' access summary
   map<Key, unsigned> key_access_summary;
+  // keep track of the hot keys' access summary
```

Comment:
These comments don't add any value - they state what the code does not why or how etc.

Reply:
+1